system-config/vagrant-reload.rb

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2013 Aidan Nagorcka-Smith
+#
+# MIT License
```

Comment:
Yeah, the copyright/license text was not in the source code, but was in the repo. To avoid confusion I think it makes sense to add it to the code we copied. See https://github.com/aidanns/vagrant-reload